app/views/users/index.html.erb

```diff
@@ -0,0 +1,50 @@
+<div class="container">
+  <p id="notice"><%= notice %></p>
+
+
+  <div class="row">
+    <%= form_with method: :get do |f| %>
+      <div class="container mt-4">
+        <h1 class='text-center'>Users</h1>
+
+        <div class="input-group">
+          <%= f.text_field :query, placeholder: 'Search', class: 'form-control rounded' %>
+          <%= f.submit 'search', data: { disabled: false }, class: 'btn btn-outline-primary' %>
+        </div>
+      </div>
+    <% end %>
+    <%= render 'search_user', users: @search_users %>
+  </div>
+
+  <%
+=begin%>
+ <div class='table-responsive d-flex justify-content-center'>
+    <table class='table table-borderless lead w-75 mt-3'>
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Surname</th>
+          <th>Email</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        <% @users.each do |user| %>
+          <tr>
+            <td><%= user.first_name %></td>
+            <td><%= user.last_name %></td>
+            <td><%= user.email %></td>
+          </tr>
+        <% end %>
+      </tbody>
+    </table>
+  </div> 
+<%
+=end%>
+
+  <br>
+
+  <div class='d-flex justify-content-center'>
+    <%= link_to 'New User', new_user_path, class: 'btn btn-success btn-lg px-3 mb-3 lead' %>
+  </div>
+</div>
```

Comment:
where is empty line?)) any file must exists empty line in the end

Reply:
resolve it

Reply:
done